__tests__/redux/actions/getArticles.test.js

```diff
@@ -176,15 +306,46 @@ describe('Reducer test', () => {
     user: JSON.parse(localStorage.user || '{}'),
   };
   it('Should return an object', () => {
-    const login = getLoginReducer(initialState4, { type: LOGIN_ERROR, payload: {} });
-    expect(typeof (login)).toBe('object');
+    const login = getLoginReducer(initialState4, {
+      type: LOGIN_ERROR,
+      payload: {},
+    });
+    expect(typeof login).toBe('object');
   });
   it('Should return an object', () => {
-    const login = getLoginReducer(initialState4, { type: LOGIN_PENDING, payload: {} });
-    expect(typeof (login)).toBe('object');
+    const login = getLoginReducer(initialState4, {
+      type: LOGIN_PENDING,
+      payload: {},
+    });
+    expect(typeof login).toBe('object');
```

Comment:
@rukundoeric not only you have to test the data types  but also you need to test the actual data.